contracts/Bond.sol

```diff
@@ -313,36 +315,44 @@ contract Bond is
     }
 
     /// @inheritdoc  IBond
-    function collateralBalance() public view returns (uint256) {
-        return IERC20Metadata(collateralToken).balanceOf(address(this));
+    function collateralBalance()
+        public
+        view
+        returns (uint256 collateralTokens)
+    {
+        collateralTokens = IERC20Metadata(collateralToken).balanceOf(
+            address(this)
+        );
     }
 
     /// @inheritdoc IBond
-    function isFullyPaid() public view returns (bool) {
-        return paymentBalance() >= totalSupply();
+    function isFullyPaid() public view returns (bool isPaid) {
```

Comment:
these seem weirdly redundant here. is there a reason for having these named?

I think i'd prefer this return variable to be called `isFullyPaid`, but then does it shadow the function name?